### PR TITLE
[WNMGDS-2154] Release 6 blog post

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

https://jira.cms.gov/browse/WNMGDS-2154

Here's the blog post contents. Any further edits we want to do can be done through PR review.

## How to test

`yarn start` or [demo link, which will not always be up-to-date](https://cmsgov.github.io/design-system/branch/pwolfert/blog-release-6/blog/release-6.0/)

### CODE REVIEW

**Review by forrestbaer — APPROVED** [review]
Looks good, I looked it over this morning and didn't have any additions or changes here. Thank you for doing this work.

File: packages/docs/content/blog/release-6.0.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+<div className="ds-u-measure--wide ds-u-text-align--center ds-u-margin-y--3">
+  <img src={scssToCssImg} alt="Sass to CSS" className="ds-u-display--inline-block" height="100" />
+</div>
+
+With this move, [as we mentioned in our previous blog post](https://design.cms.gov/blog/?theme=core#sasscss-changes), we've decided to stop distributing a Sass version of our stylesheets in our npm package, but we are still providing Sass files with the Sass versions of all our theme variables while teams transition over to using CSS Custom Properties. You can [read about how to use the CSS files and our theme variables](https://design.cms.gov/getting-started/for-developers/) over in our getting-started guide for developers.
```
Comment (line47) [inline packages/docs/content/blog/release-6.0.mdx:21]:
```suggestion
With this move, [as we mentioned in our previous blog post](https://design.cms.gov/blog/?theme=core#sasscss-changes), we've decided to stop distributing a Sass version of our stylesheets in our npm package, but we are still providing Sass files with the Sass versions of all our theme variables while teams transition over to using CSS Custom Properties. You can [read about how to use the CSS files and our theme variables](https://design.cms.gov/getting-started/for-developers/) in our getting-started guide for developers.
```

File: packages/docs/content/blog/release-6.0.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+<div className="ds-u-measure--wide ds-u-text-align--center">
+  <img src={ssnInput} alt="Animated screen capture of inputting a Social Security Number using the new label-mask pattern" className="ds-u-display--inline-block" />
+</div>
+
+Hiding the SSN is previously something application teams had to implement themselves in order to use the SSN field and maintain their users' privacy. You can [read more about the label-mask pattern](https://design.cms.gov/components/text-field/label-masked-field/)—what it is and why we built it—on the new documentation page.
```
Comment (line47) [inline packages/docs/content/blog/release-6.0.mdx:31]:
```suggestion
Hiding the SSN is previously something application teams had to implement themselves to use the SSN field and maintain their users' privacy. You can [read more about the label-mask pattern](https://design.cms.gov/components/text-field/label-masked-field/)—what it is and why we built it—on the new documentation page.
```

File: packages/docs/content/blog/release-6.0.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+### Sunsetting InVision DSM
+
+Back in 2020 we started using InVision DSM to capture specific documentation and guidance in addition to the documentation sites. InVision DSM allowed for more non-technical users to contribute documentation and guidance to the design system. The DSM solved one problem of allowing team members an easy way to create and update content. In addition, the DSM has proved that making it easier to update and maintain content using a WYSIWYG type editor is helpful for the design system. We made a lot of good progress in collaborating on specific Healthcare and Medicare guidance for product teams.
```
Comment (line47) [inline packages/docs/content/blog/release-6.0.mdx:35]:
```suggestion
Back in 2020, we started using InVision DSM to capture specific documentation and guidance in addition to the documentation sites. InVision DSM allowed for more non-technical users to contribute documentation and guidance to the design system. The DSM solved one problem of allowing team members an easy way to create and update content. In addition, the DSM has proved that making it easier to update and maintain content using a WYSIWYG-type editor is helpful for the design system. We made a lot of good progress in collaborating on specific Healthcare and Medicare guidance for product teams.
```

File: packages/docs/content/blog/release-6.0.mdx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+However, the DSM also created divergence in several important areas:  
+  
+- Child design systems didn't inherit content from core
+    - Contributors were writing the same content multiple times
+    - Product teams need to visit multiple sites to consume necessary guidance
+- DSM guidance and code library fell out of sync
+- Two different groups were responsible for updating two different doc sites (devs & designers)
+- Engineers favored GitHub-based doc site for its parity with code, while designers looked to the DSM for up-to-date guidance
+
+For these reasons we decided to migrate all the content from the DSM into our documentation site design.cms.gov and with this release have completed that effort and are sunsetting the DSM! This moves us closer toward our objective of having a single source of truth for the design system. We will continue to work on sunsetting related legacy sites like assets.cms.gov and styleguide.healthcare.gov in the future.
```
Comment (line47) [inline packages/docs/content/blog/release-6.0.mdx:46]:
```suggestion
For these reasons we decided to migrate all the content from the DSM into our documentation site design.cms.gov and with this release have completed that effort and are sunsetting the DSM! This moves us closer to our objective of having a single source of truth for the design system. We will continue to work on sunsetting related legacy sites like assets.cms.gov and styleguide.healthcare.gov in the future.
```

File: packages/docs/content/blog/release-6.0.mdx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+### System Color tokens
+
+In our [3.6.0 release](https://github.com/CMSgov/design-system/releases/tag/%40cmsgov%2Fdesign-system%403.6.0) we established design tokens for the design system including a system color tokens. System color tokens are the complete set of color tokens from which themes are built.
```
Comment (line47) [inline packages/docs/content/blog/release-6.0.mdx:50]:
```suggestion
In our [3.6.0 release](https://github.com/CMSgov/design-system/releases/tag/%40cmsgov%2Fdesign-system%403.6.0) we established design tokens for the design system including system color tokens. System color tokens are the complete set of color tokens from which themes are built.
```

File: packages/docs/content/blog/release-6.0.mdx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+<div className="ds-u-measure--wide ds-u-text-align--center">
+  <img src={swatches} alt="Color ramps from the 'System Color Tokens' page, showing a range of light to dark shades of a color" className="ds-u-display--inline-block" width="400" />
+</div>
+
+In this release we've added a page to the design system to show all color tokens in an easy to read and scannable way. We're exposing this base set of tokens to better highlight the design decisions made within a particular theme. Think of them as [a complete box of crayons](https://design.cms.gov/foundation/system-color-tokens/?theme=core). Our themes make use of certain colors from the crayon box but not all of them.
```
Comment (line47) [inline packages/docs/content/blog/release-6.0.mdx:56]:
```suggestion
In this release we've added a page to the design system to show all color tokens in an easy-to-read and scannable way. We're exposing this base set of tokens to better highlight the design decisions made within a particular theme. Think of them as [a complete box of crayons](https://design.cms.gov/foundation/system-color-tokens/?theme=core). Our themes make use of certain colors from the crayon box but not all of them.
```

File: packages/docs/content/blog/release-6.0.mdx [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+### Streamlining the ZIP Code design pattern
+
+We dove deep into a dozen CMS, HealthCare, and Medicare products, what we discovered was truly eye-opening. We found a wide range of design patterns used for ZIP code input fields, but we also found some commonalities that can lead to a more efficient and user-friendly experience.
```
Comment (line47) [inline packages/docs/content/blog/release-6.0.mdx:60]:
```suggestion
We dove deep into a dozen CMS, HealthCare, and Medicare products, and what we discovered was truly eye-opening. We found a wide range of design patterns used for ZIP code input fields, but we also found some commonalities that can lead to a more efficient and user-friendly experience.
```

File: packages/docs/content/blog/release-6.0.mdx [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+
+### Sketch layer order
+
+For many people, the New Year inspires lofty goals that, through time, eventually get neglected.
+
+Designers can fall into that same trap. We want to label and organize every layer in a file but, eventually, cracks emerge.
+
+In this non-breaking change, we shook the proverbial rug out in all three UI Kits, neatly re-aligned their categories, and listed layers alphabetically from top to bottom.
+
+Oh, and we added some fancy new thumbnails to the files to help visually separate them in Sketch cloud! Checkout our “New Years Fresh Scent” [Design System 3.0.1 UI Kits](https://www.sketch.com/workspace/7688a8c5-c5a5-4944-bd1d-81f2169122ef/p/e0c14ddc-de5f-42bb-815d-be8ba96c39d1).
```
Comment (line47) [inline packages/docs/content/blog/release-6.0.mdx:74]:
```suggestion
Oh, and we added some fancy new thumbnails to the files to help visually separate them in Sketch cloud! Check out our “New Years Fresh Scent” [Design System 3.0.1 UI Kits](https://www.sketch.com/workspace/7688a8c5-c5a5-4944-bd1d-81f2169122ef/p/e0c14ddc-de5f-42bb-815d-be8ba96c39d1).
```

File: packages/docs/content/blog/release-6.0.mdx [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+
+## What's on deck
+
+### Future font updates
+
+The design system team is turning its attention towards improving our font offerings. Our goals in this initiative are to…
+- Allow font sizes to adapt to user preferences
+- Improve performance with a variety of operating systems under poor network conditions
+- Treat fonts and their utilities with more consistency across our design system themes
+- Streamline the design process and expedite decision making as it relates to fonts
```
Comment (line47) [inline packages/docs/content/blog/release-6.0.mdx:84]:
```suggestion
- Streamline the design process and expedite decision-making as it relates to fonts
```

**Comment by pwolfert** [comment]
I'm going to merge this once the tests pass

**Pull request merged**